.github/ISSUE_TEMPLATE.md

```diff
@@ -13,9 +13,7 @@
 
 **Tell us about your environment**
 
-* **ESLint Version:**
-* **Node Version:**
-* **npm Version:**
+paste the results of **eslint --env-info** here.
```

Comment:
I think we should leave the current text and put a note in a markdown comment that lets people know about the new flag. A lot of folks are still going to be on older versions of ESLint that don't have this flag.

```md
<!--
    If you are using ESLint v6.5.0 or later, you can run ESLint with the `--env-info` flag and paste the output here.
-->

* **ESLint Version:**
* **Node Version:**
* **npm Version:**
```